src/WebJobs.Extensions.DurableTask/Listener/OutOfProcOrchestrationShim.cs

```diff
@@ -38,6 +39,8 @@ private enum AsyncActionType
             WaitForExternalEvent = 6,
             CallEntity = 7,
             CallHttp = 8,
+            SignalEntity = 9,
+            Lock = 10,
```

Comment:
Let's wait on Lock until we can fully support it.

Reply:
Understood, will remove it